src/test/ui/rfc-2497-if-let-chains/issue-83274.rs

```diff
@@ -0,0 +1,7 @@
+// check-fail
```

Comment:
Can we add another copy of this file with `// check-pass` and the gate enabled to catch any regressions?